assets/scss/_card.scss

```diff
@@ -0,0 +1,97 @@
+.card-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  @media all and (max-width: 768px) {
+    flex-direction: column;
+  }
+
+  li {
+    flex: 0 0 45%;
+    margin-bottom: 6rem;
+  }
+
+  li:nth-child(odd):last-child { margin-bottom: 0; }
+
+  .card-small {
+    padding: 2rem;
+
+    background-color: $bg-color;
+    border: 1px solid $alt-bg-color;
+    border-radius: 1.5rem;
+  }
+
+  .card {
+    display: inline-flex;
+  
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    
+    background-color: $bg-color;
+    border: 1px solid $alt-bg-color;
+    border-radius: 1.5rem;
+  
+    .card-image {
+      height: 200px;
+      margin: auto;
+    }
+    
+    .card-image[src*="jpg"] {
```

Comment:
I don't get it. Why this specific jpg rule?

Reply:
It had a jpg image for a project, without it, we can see the white background around the image.
It's not perfect because then the image is stretched...

Reply:
And this happens only with jpg files?
I don't know if understood well. Could you show me an example?

Reply:
JPG image:
With this rule:

![image](https://user-images.githubusercontent.com/35581688/82685909-3e52fa00-9c55-11ea-9559-8343590c44c4.png)

Without:

![image](https://user-images.githubusercontent.com/35581688/82685970-575bab00-9c55-11ea-8081-d585132619f4.png)

Reply:
PNG Image: (hugo-coder-featured.png)

![image](https://user-images.githubusercontent.com/35581688/82686487-36e02080-9c56-11ea-83d2-38a07c5c1e66.png)

An other one:

![image](https://user-images.githubusercontent.com/35581688/82686922-e61cf780-9c56-11ea-826e-c06eae7e4efd.png)

Looks like the border-radius property would be necessary.

I tried different types and sizes, it depends on the background color, the image ratio...
If anyone has a better approach, let me know 😉.